src/components/ProjectTagsEditor.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  MAX_PROJECT_TAGS,
+  ProjectTag,
+  projectTagOptions,
+} from 'models/project-tags'
+import { useCallback, useEffect, useState } from 'react'
+
+import ProjectTagElem from './ProjectTagElem'
+import ProjectTagsRow from './ProjectTagsRow'
+
+export default function ProjectTagsEditor({
+  initialValue,
+  onChange,
+}: {
+  initialValue?: ProjectTag[]
+  onChange?: (tags: ProjectTag[]) => void
+}) {
+  const [value, setValue] = useState<ProjectTag[]>([])
+
+  useEffect(() => {
+    if (initialValue) setValue(initialValue)
+  }, [initialValue])
```

Comment:
**question** hopefully we can find a way to avoid this useEffect cc @wraeth-eth

Reply:
please. this feels hacky but the only way I could get this to work. been so long since I played with any FormItems I may just be missing something